### PR TITLE
Pantheon updates 2021-09-22

### PR DESCRIPTION
**Pull request by bobby285271**
###### Motivation for this change

Part of Pantheon 6.0.2 updates (previous one at https://github.com/NixOS/nixpkgs/pull/138874).

###### Things done

- [x] Rebuilt NixOS to test the change.
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bobby285271** [comment]
#### pantheon.sideload

https://github.com/elementary/sideload/releases/tag/6.0.2
https://github.com/elementary/sideload/compare/6.0.1...6.0.2

> - Fix blurry app icon in multitasking view
> - Updated translations

#### pantheon.elementary-greeter

https://github.com/elementary/greeter/releases/tag/6.0.1
https://github.com/elementary/greeter/compare/6.0.0...6.0.1

> Improvements:
> - Hide clock during initial setup
> - Use selected accent color for logged in checkmark
>
> Minor updates:
> - Updated translations

#### pantheon.elementary-calendar

https://github.com/elementary/calendar/releases/tag/6.0.2
https://github.com/elementary/calendar/compare/6.0.1...6.0.2

> Fixes:
> - Fix notification icons in the notification indicator
> - Fix an issue where event timezones were overwritten
> - Ensure Calendar is launched when notifications are clicked
> 
> Minor Updates:
> - Updated translations

#### pantheon.switchboard-plug-keyboard

https://github.com/elementary/switchboard-plug-keyboard/releases/tag/2.5.1
https://github.com/elementary/switchboard-plug-keyboard/compare/2.5.0...2.5.1

> Minor updates:
> - Ensure IBus automatically starts on login when needed
> - Updated translations

#### pantheon.switchboard-plug-mouse-touchpad

https://github.com/elementary/switchboard-plug-mouse-touchpad/releases/tag/6.1.0
https://github.com/elementary/switchboard-plug-mouse-touchpad/compare/6.0.0...6.1.0

> New features:
> - New toggle for “Double-tap and move to drag”
>
> Minor updates:
> - Updated translations

**Comment by bobby285271** [comment]
https://github.com/elementary/switchboard-plug-keyboard/commit/c9dc4a83ee52c67e090969a96e0cb6ed322a3a9f#diff-42c5c427cac19224e91a4407f2cc48c60941a3d15f7a4e019a4e97cdd4b51ad4R376

Probably needs a fix path here.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 139057` at 6632becb run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built successfully:</summary>
  <ul>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.sideload</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
  </ul>
</details>

File: pkgs/desktops/pantheon/apps/switchboard-plugs/mouse-touchpad/default.nix [hunk 1]
```diff
@@ -60,12 +60,6 @@ stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       touchegg = touchegg;
     })
-    # Upstream code not respecting our localedir
-    # https://github.com/elementary/switchboard-plug-mouse-touchpad/pull/185
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/desktops/pantheon/apps/switchboard-plugs/mouse-touchpad/default.nix:65]: Please remove the unused fetchpatch from inputs.

File: pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix [hunk 2]
```diff
@@ -89,12 +89,6 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
-    # Upstream code not respecting our localedir
-    # https://github.com/elementary/greeter/pull/545
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix:94]: Please remove the unused fetchpatch from inputs.

File: pkgs/desktops/pantheon/apps/switchboard-plugs/mouse-touchpad/default.nix (hunk 1 above)
Comment (bobby285271) [reply pkgs/desktops/pantheon/apps/switchboard-plugs/mouse-touchpad/default.nix:65]: https://github.com/NixOS/nixpkgs/pull/139163

File: pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix (hunk 2 above)
Comment (bobby285271) [reply pkgs/desktops/pantheon/desktop/elementary-greeter/default.nix:94]: https://github.com/NixOS/nixpkgs/pull/139163

**Pull request merged**